tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -118,15 +118,10 @@ def run_pipeline_test(self, speech_recognizer, examples):
                 },
             )
         else:
-            # Non CTC models cannot use chunk_length
-            with self.assertRaises(ValueError) as v:
-                outputs = speech_recognizer(audio, chunk_length_s=10)
-            self.assertEqual(v.exception, "")
```

Comment:
PR #20104 have the following change (see [here](https://github.com/huggingface/transformers/commit/25c451e5a044969eb91e1e481574a2bfca5130ca#diff-ccc8e98fcaa81fdf6317a652438a309bcede0bbe336774288c2fcf91d9f11082)), and we don't expect to have exception here anymore

from
```python
        if chunk_length_s:
            if self.type not in {"ctc", "ctc_with_lm"}:
                raise ValueError(
                    "`chunk_length_s` is only valid for CTC models, use other chunking options for other models"
                )
```
to
```python
            if self.type == "seq2seq" and not ignore_warning:
                logger.warning(
                    "Using `chunk_length_s` is very experimental with seq2seq models. The results will not necessarily"
                    " be entirely accurate and will have caveats. More information:"
                    " https://github.com/huggingface/transformers/pull/20104. Ignore this warning with pipeline(...,"
                    " ignore_warning=True)"
                )
```